pallets/migrations/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "pallet-migrations"
+version = "0.1.0"
+authors = ["PureStake"]
+edition = "2018"
+description = "migrations management pallet"
+
+[dependencies]
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
+log = "0.4"
+sp-std = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
+parity-scale-codec = { version = "2.0.0", default-features = false }
+environmental = { version = "1.1.2", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
```

Comment:
these could be dev-dependencies and then you can remove the `default-features=false` and also remove them from `std` features further below

Reply:
Good point. Fixed in d9e561b